modules/react-native-status/android/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
     compile 'com.github.ericwlange:AndroidJSCore:3.0.1'
     compile 'status-im:function:0.0.1'
 
-    String statusGoVersion = 'develop-gd71c66a2'
+    String statusGoVersion = 'junk-log-NotifyUsers-g605a2123-62'
```

Comment:
can't we use current `status-go`'s `develop` here?

Reply:
This is just a temporary commit to gain additional logging on the go side to allow QA to verify.